features/glance.feature

```diff
@@ -0,0 +1,39 @@
+@glance
+Feature: Glance
+  As an administrator
+  I want to make sure that Glance component is working correctly
+
+  @create_kvm_image
```

Comment:
Just curious what the point of this tag is?  I didn't think tags were intended to be used for labelling individual scenarios, but more for tagging scenarios which have a particular characteristic (e.g. `@slow` or `@kvm`).